consensus/src/ordered_broadcast/engine.rs

```diff
@@ -325,15 +318,26 @@ impl<
                 },
 
                 // Handle refresh epoch deadline
-                _ = refresh_epoch => {
-                    debug!("refresh epoch");
-                    // Simply continue; the epoch will be refreshed on the next iteration.
+                epoch = epoch_updates.next() => {
+                    // Error handling
+                    let epoch = match epoch{
```

Comment:
Is this linted? I usually feel like there would be a space between `epoch{`

Reply:
I think these macros are really messing with the linter...